server/plugin/command.go

```diff
@@ -265,7 +270,53 @@ func (p *Plugin) handleSubscriptionsList(_ *plugin.Context, args *model.CommandA
 	return txt
 }
 
+func (p *Plugin) checkIfConfiguredWebhookExists(githubClient *github.Client, repo, owner string, ctx context.Context) (bool, error) {
+	found := false
+	opt := &github.ListOptions{
+		PerPage: GithubListOptionsPerPageValue,
+	}
+	siteURL := *p.client.Configuration.GetConfig().ServiceSettings.SiteURL
+
+	for {
+		var githubHooks []*github.Hook
+		var githubResponse *github.Response
+		var err error
+
+		if repo == "" {
+			githubHooks, githubResponse, err = githubClient.Organizations.ListHooks(ctx, owner, opt)
+		} else {
+			githubHooks, githubResponse, err = githubClient.Repositories.ListHooks(ctx, owner, repo, opt)
+		}
+
+		if err != nil {
+			p.API.LogWarn("Not able to get the list of webhooks", "Owner", owner, "Repo", repo, "Error", err.Error())
+			// Breaking from the loop if the repo or org is not found
+			if githubResponse.StatusCode == 404 && strings.Contains(err.Error(), "404 Not Found") {
+				found = true
+				break
```

Comment:
I still don't get why the code returns a `true` here. Should it return  `false, nil` here?

Reply:
Can you help me understand this code?

Reply:
In case we are subscribing to a username then we will get the 404 error. I have handled that case that we are not going to show any message for this particular.

Reply:
yes @hanzei. It is just my suggestion but i think we should just log this warning in the server logs rather than showing this warning to the user. Please give your views on this

Reply:
I'm 2/5 that we should show a warning to the user as they expect the webhooks to work. For orgs and their repos the check just works. It's surprising for a user if this special case doesn't properly check.

Reply:
sure @hanzei will make this change and add the warnings as needed